Content.Server/Radio/EntitySystems/JammerSystem.cs

```diff
@@ -38,19 +38,27 @@ public override void Update(float frameTime)
 
     private void OnActivate(EntityUid uid, RadioJammerComponent comp, ActivateInWorldEvent args)
     {
-        var activated = !HasComp<ActiveRadioJammerComponent>(uid) && 
-            _powerCell.TryGetBatteryFromSlot(uid, out var battery) &&
+        var active = HasComp<ActiveRadioJammerComponent>(uid);
+        var charged = _powerCell.TryGetBatteryFromSlot(uid, out var battery) &&
             battery.CurrentCharge > comp.Wattage;
-        if (activated)
+
+        var state = Loc.GetString(!active ? "radio-jammer-component-on-state" : "radio-jammer-component-off-state");
+        var message = Loc.GetString("radio-jammer-component-on-use", ("state", state));
+
+        if(!charged)
+        {
+          message = Loc.GetString("radio-jammer-component-no-charge");
+        }
+
+        if (!active && charged)
         {
             EnsureComp<ActiveRadioJammerComponent>(uid);
         }
         else
         {
             RemComp<ActiveRadioJammerComponent>(uid);
         }
```

Comment:
Can this not use PowerCellDraw?

Reply:
> Can this not use PowerCellDraw?
I have not learnt the code base enough to know what that is.